users/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import re
+import requests
 
-# Create your views here.
+from twilio.rest  import Client
+from django.views import View
+from django.http  import JsonResponse
+from random       import randint
+from users.models import User, PhoneCheck
+from my_settings  import ACCOUNT_SID, AUTH_TOKEN
+
+class SendSmSView(View):
+    def post(self, request):
+        data         = json.loads(request.body)
+        input_mobile = data.get('mobile')
+        try:
+            if not input_mobile:
+                return JsonResponse({'message': 'KEY_ERROR'}, status=401)
+            if re.match(r"(010)\d{4}\d{4}",input_mobile):
+                to_mobile = '+82'+ input_mobile[1:]
+            else:
+                return JsonResponse({'message':'INVALID_PHONE_NUMBER'}, status=401)
+            
+            check_number = randint(1000,9999)
+
+            if PhoneCheck.objects.filter(check_id=input_mobile).exists():
+                PhoneCheck.objects.filter(check_id=input_mobile).update(check_number=check_number)
+            else:
+                PhoneCheck.objects.create(check_id=input_mobile, check_number=check_number)
+
+            client       = Client(ACCOUNT_SID, AUTH_TOKEN)
+            client.messages.create(
+                body  =f'milliem의사서 인증을 위해 [{check_number}]을 입력해주세요.' ,
+                from_ ='+14078716367',
+                to    = to_mobile
+            )
+            return JsonResponse({'message': 'SUCCESS'}, status=200)
+        except:
```

Comment:
```suggestion
        except KeyError: 
```
위처럼 except만 선언하면 모든 익셉션을 키에러로 처리하게됩니다. 
또 지금의 상황에선 딱히 딕셔너리 키호출을 하는 코드가 없는데, 이 익셉션을 해야하는지 생각해보세요.

Reply:
수정 진행 하였습니다.